common/src/main/java/com/google/udmi/util/MessageUpgrader.java

```diff
@@ -127,42 +123,47 @@ private Object upgradeRaw(boolean forceUpgrade) {
 
     if (minor == 4 && patch < 1) {
       JsonNode before = message.deepCopy();
-      upgrade_1_4_1();
+      upgradeTo_1_4_1();
       upgraded |= !before.equals(message);
       patch = 1;
     }
 
     if (upgraded && message.get(VERSION_KEY) != null) {
       message.put(UPGRADED_FROM, originalVersion);
       message.put(VERSION_KEY, String.format(TARGET_FORMAT, major, minor, patch));
+    } else {
+      // Files are now updated to Current version of UDMI
+      if(message.has(VERSION_KEY)){
```

Comment:
lint won't like this?  Should have space after "if"

Also, comment is unclear or superfluous -- it's not particularly clarifying the why of anything?